app/models/customer.rb

```diff
@@ -0,0 +1,13 @@
+class Customer < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :phone, presence: true
+  validates :phone, numericality: { only_integer: true }
+  validates :phone, length: { is: 10 }
+  validates :email, presence: true
```

Comment:
This looks good, but just so you know you can combine validations of the same type into a single line. For example here you could do:

`validates :first_name, :last_name, :phone, :email, presence: true` and it would work the same.

Reply:
Thanks Neel. I guess I was in a rush to catch up with the curriculum so I neglected to check out the RubyDoc on relevant topics along the way. Will edit my code now and revisit the lesson to read up on the doc.